gallery/src/components/NxTable/NxTableClickableExample.tsx

```diff
@@ -32,6 +32,19 @@ const NxTableClickableExample = () => {
             <NxTable.Cell chevron/>
           </NxTable.Row>
         )}
+        <NxTable.Row isClickable onClick={() => alert('Clicked last row')} clickAccessibleLabel="The last row">
+          <NxTable.Cell>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed sed lorem vitae dolor mattis imperdiet a ac
+            nisl. Maecenas quis dapibus lacus, non lacinia dolor. In hac habitasse platea dictumst. Sed sit amet
+            nulla facilisis, elementum tellus quis, mattis turpis. Vivamus accumsan, purus vel maximus condimentum,
+            augue quam hendrerit velit, sed finibus nulla justo sit amet est. Nam tincidunt efficitur dapibus.
+            Quisque et pellentesque ante. Sed porttitor sem ipsum, sit amet blandit felis hendrerit nec. Suspendisse
+            congue, tortor tristique sollicitudin consectetur, sapien purus posuere massa, eget finibus nulla ipsum
+            sed sapien.
+          </NxTable.Cell>
+          <NxTable.Cell>Content</NxTable.Cell>
+          <NxTable.Cell chevron/>
```

Comment:
In Safari the Chevrons appear as buttons. 
<img width="1119" alt="Screen Shot 2021-04-28 at 10 58 18 AM" src="https://user-images.githubusercontent.com/1991443/116426026-c308a980-a810-11eb-95bc-90002947d325.png">

Reply:
Looks like you must override the UA styes with `border: none;` and `background-colour: transparent;`

Reply:
> In Safari the Chevrons appear as buttons.

Checked on two computers, both running Safari 14.1 and I can lo longer reproduce.